### PR TITLE
Update tag.mdx

### PR DESCRIPTION
**Pull request by vavalos5**
Transferring content from Figma to Tag page. Edits still coming.

[Deploy Preview](https://deploy-preview-4079--keen-mayer-a86c8b.netlify.app/components/tag)

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

**Comment by SeamusLeonardHPE** [comment]
@vavalos5 @britt6612 

Hey Brittany - 
I'm afraid of breaking this PR - When you have time can you update this page to match the content laid out here:

https://www.figma.com/design/IiBz42hz6zRVC9g2TFMlTU/%234033-TAGS-Documentation?node-id=2-7078&t=UlDsW0hMCpnrQvUo-4

**Comment by britt6612** [comment]
In order to have the type highlight we need to make an enhancement to grommet 
https://github.com/grommet/grommet/issues/7317

File: aries-site/src/pages/components/tag.mdx [hunk 1]
```diff
@@ -1,144 +1,123 @@
...
 
-Tags do not receive a background color. Instead, tags inherit the background of their container.
+### Drawing attention
+Highlight tags can be used to draw attention to an element on the page.
+They are a decorative variant of the static tag.
+Highlights should only contain descriptions that are temporary, such as “New”,
+“Updated”. As such they should not be used a permanent design feature.
 
-## Name-value pair tags
+<Example
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx:59]: done

File: aries-site/src/pages/components/tag.mdx [hunk 2]
```diff
@@ -1,144 +1,123 @@
...
+  previewWidth="large"
+  height={{ min: 'small' }}
+>
+  <TagAnatomy />
+</Example>
 
-The experience for creating a Name-Value pair tag occurs in 2 steps: Create the name and then create the value. Alternatively, select an existing name and create a new value.
 
-## Assigning tags
+| Label | Region        | Purpose                                                      | Required | Notes                                                       |
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx:78]: done

File: aries-site/src/pages/components/tag.mdx [hunk 3]
```diff
@@ -1,144 +1,123 @@
...
-  docs="https://v2.grommet.io/tag?theme=hpe#props"
-  showResponsiveControls={['laptop', 'mobile']}
-  template
->
-  <CreateTagSimple />
-</Example>
+### Tag 
+- Contains simple, concise information used to annotate, categorize, group or otherwise label items.
+- Used to display secondary or supplementary information related to an object or item. 
+- Can be used to communicate the state or condition of something (e.g. published, deployed, new, updated, etc.). State is defined here.
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx]: done

File: aries-site/src/pages/components/tag.mdx [hunk 4]
```diff
@@ -1,144 +1,123 @@
...
+- A tag should not be used to convey system status, that is considered primary information.
 
-### Assigning name-value tags
+### Static 
+- All points as above plus...
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx]: done

File: aries-site/src/pages/components/tag.mdx [hunk 5]
```diff
@@ -1,144 +1,123 @@
...
-## Filtering with tags
+- A removable tag is interactive via the icon button.
+- Can be used to dismiss a category/value set from a filtered view.
+- Can be used to remove a value that has been assigned to an item or resource.
 
-Filtering with tags allows the user to search, select, and remove tags from a set of filter criteria in order to find relevant data.
+### Highlight
 
-### Filtering with tags on a detail page
+- A highlight tag used to indicate a temporary or transient state associated with an item - eg new, updated, promotional
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx]: done

File: aries-site/src/pages/components/tag.mdx [hunk 6]
```diff
@@ -1,144 +1,123 @@
...
-  <TagBasicAnatomy />
+<Example
+  componentName="Tag"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/select/SelectExample.js"
+  docs="https://v2.grommet.io/tag?theme=hpe"
+  figma=""
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Tag/Tag.js"
+>
+  <TagSimple />
 </Example>
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx:19]: done

File: aries-site/src/pages/components/tag.mdx [hunk 7]
```diff
@@ -1,144 +1,123 @@
...
-## Filtering with tags
+- A removable tag is interactive via the icon button.
+- Can be used to dismiss a category/value set from a filtered view.
+- Can be used to remove a value that has been assigned to an item or resource.
 
-Filtering with tags allows the user to search, select, and remove tags from a set of filter criteria in order to find relevant data.
+### Highlight
 
-### Filtering with tags on a detail page
+- A highlight tag used to indicate a temporary or transient state associated with an item - eg new, updated, promotional.
```
Comment (vavalos5) [inline aries-site/src/pages/components/tag.mdx:110]: e.g. new,

File: aries-site/src/pages/components/tag.mdx [hunk 8]
```diff
@@ -1,144 +1,122 @@
...
+  <TagSimple />
 </Example>
 
-1. Border
-1. Value
+## Use cases
 
-Tags can have additional elements depending on the tags’ intended use and context:
+### Displaying metadata
+[Static tags](/components/tag#static) can be used to display non critical concise information with
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:24]:
```suggestion
[Static tags](/components/tag#static) can be used to display non-critical, concise information with
```

File: aries-site/src/pages/components/tag.mdx [hunk 9]
```diff
@@ -1,144 +1,122 @@
...
-- Name: up to 128 characters
-- Value: up to 256 characters
-
-### Remove button
-
-Applying the remove button (see element #4 in the [anatomy figure](#anatomy)) is dependent upon the use case and not a requirement.
-
-The remove tag button may be used when:
+### Managing metadata
+Tags can be created, assigned and [removed](/components/tag#removable) from resources/items by a user.
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:35]:
```suggestion
Tags can be created, assigned and [removed](/components/tag#removable) from resources or items by a user.
```

File: aries-site/src/pages/components/tag.mdx [hunk 10]
```diff
@@ -1,144 +1,122 @@
...
+>
+  <TagResource />
+</Example>
 
-Tags do not receive a background color. Instead, tags inherit the background of their container.
+### Drawing attention
+[Highlight tags](/components/tag#highlight) can be used to draw attention to an element on the page.
+They are a decorative variant of the [static tag](/components/tag#static).
+Highlights should only contain descriptions that are temporary, such as “New”,
+“Updated”. As such they should not be used a permanent design feature.
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:57]:
```suggestion
“Updated”. As such they should not be used as a permanent design feature.
```

File: aries-site/src/pages/components/tag.mdx [hunk 11]
```diff
@@ -1,144 +1,122 @@
...
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/utils.js',
-  ]}
-  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/tag/CreateTagSimple.js"
-  docs="https://v2.grommet.io/tag?theme=hpe#props"
-  showResponsiveControls={['laptop', 'mobile']}
-  template
->
-  <CreateTagSimple />
-</Example>
+### Tag
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:88]: Seems like this should just be "Static", Tag is the thing and static is the type. Then the other types build off of static.

If so, rename this to "Static" and remove the current section called "Static".

File: aries-site/src/pages/components/tag.mdx [hunk 12]
```diff
@@ -1,144 +1,122 @@
...
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/utils.js',
-  ]}
-  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/tag/CreateTag"
-  docs="https://v2.grommet.io/tag?theme=hpe#props"
-  showResponsiveControls={['laptop', 'mobile']}
-  template
->
-  <CreateTag />
-</Example>
+### Interactive
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:97]: Are we planning to have a visual example here @britt6612? Maybe could use the `plain` treatment like anatomy diagrams

File: aries-site/src/pages/components/tag.mdx [hunk 13]
```diff
@@ -1,144 +1,122 @@
...
-  <CreateTag />
-</Example>
+### Interactive 
+- Interactive tags enable users to discover, filter, or navigate to identically tagged items.
+- An interactive tag can be actioned by mouse or keyboard.
 
+### Removable
 
-## Filtering with tags
+- A removable tag is interactive via the icon button.
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:103]:
```suggestion
- A removable tag is interactive via the "Close" icon button.
```

File: aries-site/src/pages/components/tag.mdx [hunk 14]
```diff
@@ -1,144 +1,122 @@
...
+- Can be used to dismiss a category/value set from a filtered view.
+- Can be used to remove a value that has been assigned to an item or resource.
 
-Filtering with tags allows the user to search, select, and remove tags from a set of filter criteria in order to find relevant data.
+### Highlight
 
-### Filtering with tags on a detail page
+- A highlight tag used to indicate a temporary or transient state associated with an item - eg new, updated, promotional.
+- A highlight tag draws attention but is no louder than a secondary/primary button.
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:110]:
```suggestion
- A highlight tag draws attention but is no louder than a secondary or primary button.
```

File: aries-site/src/pages/components/tag.mdx [hunk 15]
```diff
@@ -1,144 +1,122 @@
...
 
+### Removable
 
-## Filtering with tags
+- A removable tag is interactive via the icon button.
+- Can be used to dismiss a category/value set from a filtered view.
+- Can be used to remove a value that has been assigned to an item or resource.
 
-Filtering with tags allows the user to search, select, and remove tags from a set of filter criteria in order to find relevant data.
+### Highlight
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:107]: On these bullets, it feels repetitive having "A highlight tag.." can we update to just be

- Used to indicate...
- Draws attention...

File: aries-site/src/pages/components/tag.mdx [hunk 16]
```diff
@@ -1,144 +1,122 @@
...
 
-### Filtering with tags on a detail page
+- A highlight tag used to indicate a temporary or transient state associated with an item - eg new, updated, promotional.
+- A highlight tag draws attention but is no louder than a secondary/primary button.
+- A highlight tag is not interactive in any way.
+- A tags size should be limited to x-small or small.
+- Use “gradientEdge” border for highlight tags. This treatment should be used sparingly.
```
Comment (taysea) [inline aries-site/src/pages/components/tag.mdx:113]:
```suggestion
- Use a gradient border for highlight tags. This treatment should be used sparingly.
```
The term "gradientEdge" is tool specific. We should make this more generic.

File: aries-site/src/pages/components/tag.mdx (hunk 12 above)
Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx:97]: yes I was going to add I was waiting on `border` stuff but I can add!

Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx:87]: added

File: aries-site/src/pages/components/tag.mdx (hunk 15 above)
Comment (britt6612) [reply aries-site/src/pages/components/tag.mdx:119]: done

**Comment by taysea** [comment]
Once https://github.com/grommet/hpe-design-system/pull/4230 merges in here, I am happy for this PR to be merged in. Nice work!

**Review by taysea — APPROVED** [review]
LGTM. Thank you

**Pull request merged**